test_runner/performance/test_perf_pgbench.py

```diff
@@ -84,8 +84,9 @@ def run_test_pgbench(env: PgCompare, scale: int, duration: int, workload_type: P
 
     if workload_type == PgBenchLoadType.INIT:
         # Run initialize
-        options = "-cstatement_timeout=1h " + env.pg.default_options["options"]
-        init_pgbench(env, ["pgbench", f"-s{scale}", "-i", env.pg.connstr(options=options)])
+        init_pgbench(
+            env, ["pgbench", f"-s{scale}", "-i", env.pg.connstr(options="-cstatement_timeout=1h")]
+        )
```

Comment:
```suggestion
        options = "-cstatement_timeout=1h " + env.pg.default_options.get("options", "")
        init_pgbench(env, ["pgbench", f"-s{scale}", "-i", env.pg.connstr(options=options)])
```

This should fix 
```
    options = "-cstatement_timeout=1h " + env.pg.default_options["options"]
E   KeyError: 'options'
```

Reply:
Makes sense, I applied that fix. I think the ALTER DATABASE is nevertheless more straightforward than modifying the connection string so I kept the yaml file change too.